moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -393,7 +396,7 @@ bool IKFastKinematicsPlugin::computeRelativeTransform(const std::string& from, c
   }
 
   transform = robot_state->getGlobalLinkTransform(from_link).inverse() * robot_state->getGlobalLinkTransform(to_link);
-  differs_from_identity = !transform.matrix().isIdentity();
+  differs_from_identity = !transform.matrix().isIdentity(RELATIVE_TRANSFORM_TOLERANCE);
```

Comment:
Why do you think, we should change the default (machine) tolerance value here?

Reply:
Because we can avoid unnecessary computations, if we account for numerical errors. However I saw that the default precision is `1e-12`, which should be fine.